### ISSUE TITLE
nio/_compat.py is buggy for py3

### ISSUE THREAD

**Issue by AXGKl** [poljar/weechat-matrix#103]
My `/script load matrix.py` was failing with importlib error from nio package.

Bug is in nio _compat.py, this is the error I got in weechat, directly reproduced:

```
[root@fed30 nio]# pwd
/root/miniconda3/envs/weechat/lib/python3.7/site-packages/nio
root@fed30 nio]# python3 -c 'import _compat; _compat.package_installed("foo")'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/root/miniconda3/envs/weechat/lib/python3.7/site-packages/nio/_compat.py", line 27, in package_installed
    spec = importlib.util.find_spec(package_name)
AttributeError: module 'importlib' has no attribute 'util'
```

You can work around it easily by importing `importlib.util` earlier in the process, e.g. in matrix.py itself:

```
[root@fed30 nio]# python3 -c 'import importlib.util; import _compat; _compat.package_installed("foo")'
```

Cheers,
G.

**Comment by poljar** [comment]
This has been fixed by https://github.com/poljar/matrix-nio/commit/8268d47787975d0db67d77aa33aa200711253cf2 and https://github.com/poljar/matrix-nio/commit/ff1f7624c879c3dd2c510a4f44eefcb1d1586212. A nio release with the fix should be coming soon.

**Comment by poljar** [comment]
This is now fixed in nio 0.4.1.
